### ISSUE TITLE
NickServ authentication does not work

### ISSUE THREAD

**Issue by Tjzabel** [RITlug/teleirc#331]
# Summary

When connecting to an IRC channel with NickServ authentication, TeleIRC fails to authenticate.

# Standard debugging steps

## How to reproduce?

1. Fill in `IRC_NICKSERV_PASS` in the `.env`.
2. Attempt to run TeleIRC

## Expected behavior

TeleIRC is able to successfully connect to IRC channel and authenticate with NickServ

## Additional Details
It looks like Matterbridge had this issue [once-upon-a-time](https://github.com/42wim/matterbridge/commit/a2b45bc799de77aa278212271709201336e9e796).

**Comment by jwflory** [comment]
I hit this issue today while using my development environment. Freenode doesn't like my VPN and requires me to authenticate. So in order to test the bot locally, I have to turn off my VPN. Mildly annoying. It would be nice to fix this for v2.1.0.

**Comment by jwflory** [comment]
_Discussed in [2020-10-11 meeting](https://youtu.be/A5T_kpCdysw)._

---

@Zedjones is focusing on this one for the next sprint. Once this is completed, @Tjzabel will follow up with #131.

**Comment by Zedjones** [comment]
I can't immediately reproduce this on my machine. I have had issues with VPN services in the past, but I don't think there's anything we can do about that since they're just blocking a range of IPs. The only way we could get around that is if we allowed users to configure a proxy. 

But in regards to the NickServ issue, it seems to successfully log in for me.

**Comment by jwflory** [comment]
_Discussed in [2020-10-25 meeting](https://youtu.be/V4UaDtmRrfQ)_.

---

I need to retest with my connection via my VPN. I thought about this more after our meeting, and I will wait to retest until @Tjzabel completes #131. That will make it easier for me to test and triage this bug further.

**Comment by jwflory** [comment]
I am no longer able to replicate this. The issue I had before was actually because my password required escaping certain characters. So, I was actually logging in, but somewhere along the way, my password was missing a character or something. 🤷🏻 

After #355 and now being able to specify the NickServ username separately from the bot nick, I am unable to replicate. I am closing this issue out as `wontfix` and removing it from the project board. 🎬
